Resources/public/scss/modules/sub-items-list/_view.switcher.button.scss

```diff
@@ -0,0 +1,37 @@
+.c-view-switcher-btn {
+    max-width: calculateRem(30px);
+    height: calculateRem(30px);
+    padding: 0;
+    margin: 0;
+    border: calculateRem(2px) solid $ez-white;
+    background: $ez-white;
+    color: $ez-color-secondary;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+
+    &:not(.c-view-switcher-btn--active):hover,
```

Comment:
It can be further shortened to: `&:not(&--active):hover`. But I'm not sure if want to do this as it may be less readable.

Reply:
I'll keep it like this, for better readability.